string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,24 @@
+/* exported capitalizeWord */
+/* Make storage  to include characters after the first
+-make storage to only include the first character
+-check if the word is javascript
+  -if true return the output as Javascript
+-check if the word has any capital letters one by one, this will start after the first letter
+  if true add the letter to the storage and repeat the process
+  if false add the letter to the storage and repeat the process
+-once all letters have been looked at return the output capitalizing the first letter
+then lowercasing the rest of the letters
+*/
+function capitalizeWord(word) {
+  var yesSir = '';
+  var firstOne = word[0];
+  for (var i = 1; i < word.length; i++) {
+
+    if (word.toUpperCase(word) === 'JAVASCRIPT') {
+      return 'JavaScript';
+    } else if (word[i] === word.toUpperCase(i)) {
+      yesSir += word[i];
```

Comment:
You lowercase all the characters at the end anyway, you don't need this special check. Notice that the work is the same for this block and the `else` block below.